### ISSUE TITLE
[Feature Request] Notification in work time only

### ISSUE THREAD

**Issue by MonkAlex** [telegramdesktop/tdesktop#23940]
### Is your feature request related to a problem?

I have some work chats.
Notification enabled, i want get messages and react fast.

When work time end (9 am - 6 pm for example) i not interested in any notify. Not today.
Now i can just disable notifications for 4-18 hours, but need do it manual and really need 15 hours

### Describe the solution you'd like

Notification period on channel. Just set 9 am -> 6 pm and not touch this settings.

### Describe alternatives you've considered

Maybe 15 hours delay in default settings
Maybe some of Do not Disturb mode.

**Comment by RememberTheAir** [comment]
If you're using windows, I think you can use enable native notifications (Notifications Settings), and enable Windows' Notification Assistant to "snooze" the notifications you receive during a certain time interval

**Comment by MonkAlex** [comment]
> If you're using windows, I think you can use enable native notifications (Notifications Settings), and enable Windows' Notification Assistant to "snooze" the notifications you receive during a certain time interval

Yeah, think about that. But its snooze all notifications, not only work channels =)

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/151#issuecomment-63627345

**Comment by Akkarine** [comment]
I would like to configure "mute" schedule per channel/group/chat.

**Comment by Aokromes** [comment]
hackaround: https://github.com/lx-s/WinMute
